docs_src/microservices/application/V2Migration.md

```diff
@@ -0,0 +1,298 @@
+
+##### ResponseData
+
+The `SetResponseData` and `ResponseData` APIs replace the previous `Complete` function and direct access to the `OutputData` field.
+
+##### ResponseContentType
+
+The `SetResponseContentType` and `ResponseContentType` APIs replace the previous direct access to the `ResponseContentType` field.
+
+##### RetryData
+
+The `SetRetryData` API replaces the `SetRetryData` function and direct access to the `RetryData` field.
+
+##### MarkAsPushed
+
+The `MarkAsPushed` capability has been removed
+
+##### PushToCore
+
+The `PushToCore` API replaces the `PushToCoreData` function. The API signature has changed. See the [PushToCore](../AppFunctionContextAPI/#pushtocore) section for more details.
+
+##### New Capabilities
+
+Some new capabilities have been added to the new AppFunctionContext API. See the [App Function Context](../AppFunctionContextAPI) API section for complete details.
+
+## App Service Configurable Profiles
+
+Custom profiles used with App Service Configurable are configuration files. These follow the same migration above for custom  [Application Service configuration](#configuration), except for the Configurable Functions Pipeline items.  The following are the changes for the Configurable Functions Pipeline:
+
+1. `FilterByValueDescriptor` changed to `FilterByResourceName`. See the [FilterByResourceName](../AppServiceConfigurable/#filterbyresourcename) section for details.
+2. `TransformToXML` and `TransformToJSON` have been collapsed into `Transform` with additional parameters. See the [Transform](../AppServiceConfigurable/#transform) section for more details.
+3. `CompressWithGZIP` and `CompressWithZLIB` have been collapsed into `Compress` with additional parameters. See the [Compress](../AppServiceConfigurable/#compress) section for more details.
+4. `EncryptWithAES` has been changed to `Encrypt` with additional parameters. See the [Encrypt](../AppServiceConfigurable/#encrypt) section for more details.
+5. `BatchByCount`, `BatchByTime` and `BatchByTimeAndCount` have been collapsed into `Batch` with additional parameters. See the [Batch](../AppServiceConfigurable/#batch) section for more details.
+6. `SetOutputData` has been renamed to `SetResponseData`. See the [SetResponseData](../AppServiceConfigurable/#setresponsedata) section for more details.
+7. `PushToCore` parameters have changed. See the [PushToCore](../AppServiceConfigurable/#pushtocore) section for more details.
+9. `HTTPPost`, `HTTPPostJSON`, `HTTPPostXML`, `HTTPPut`, `HTTPPutJSON` and `HTTPPutXML` have been collapsed into `HTTPExport` with additional parameters. See the [HTTPExport](../AppServiceConfigurable/#httpexport) section for more details.
+10. `MQTTSecretSend` has been renamed to `MQTTExport` with additional parameters. See the [MQTTExport](../AppServiceConfigurable/#mqttexport) section for more details.
+11. `MarkAsPushed` has been removed.
+11. `MQTTSend` has been removed.
```

Comment:
Missing an `8` and have 2 `11`s. I know markdown will do the right thing here, just pointing it out. Sometimes I just use `1.` for all to avoid re-numbering, but that is personal preference.

Reply:
I use Typora for my markdown editor, which renders as you edit, so I don't see them out of order. ;-)